core/src/main/java/org/apache/struts2/StrutsConstants.java

```diff
@@ -234,6 +234,8 @@ public final class StrutsConstants {
     /** The name of the parameter to determine whether static field access will be allowed in OGNL expressions or not */
     public static final String STRUTS_ALLOW_STATIC_FIELD_ACCESS = "struts.ognl.allowStaticFieldAccess";
 
+    public static final String STRUTS_OGNL_GUARD = "struts.ognlGuard";
```

Comment:
What about `struts.ognl.guard` to keep everything in the same _namespace_?

Reply:
Hmm, the other options that are in the `ognl.` namespace are configuration options for how OGNL executes. This on the other hand is an extension point for the bean/interface named `OgnlGuard`.

Up to you, I'm not fussed either way - let me know and I'll rename it :)

Reply:
Ah... right :) We need to figure out a better naming at some point for the extension points, having that said I'm good :)